src/CRM/CivixBundle/Resources/views/Code/api4action.php.php

```diff
@@ -0,0 +1,28 @@
+<?php echo "<?php\n"; ?>
+namespace Civi\Api4\Action\<?php echo $entityNameCamel ?>;
+
+use Civi\Api4\Generic\AbstractAction;
+use Civi\Api4\Generic\Result;
+
+/**
+ *  Class <?php echo $actionNameCamel ?>.
+ *
+ * Provided by the <?php echo $extensionName ?> extension.
+ *
+ * @package Civi\Api4
+ */
+class <?php echo $actionNameCamel ?> extends AbstractAction {
+
+ /**
+  * @inheritDoc
+  *
+  * @param \Civi\Api4\Generic\Result $result
+  *
+  * @throws \API_Exception
+  * @throws \CiviCRM_API3_Exception
```

Comment:
Why would it throw an `CiviCRM_API3_Exception`?

Reply:
copy  & paste - still hitting stuff